internals/cli/cli_test.go

```diff
@@ -168,6 +169,37 @@ func (s *PebbleSuite) TestGetEnvPaths(c *C) {
 	c.Assert(socketPath, Equals, "/path/to/socket")
 }
 
+func (s *PebbleSuite) TestGetAdditionalAdminUIDs(c *C) {
+	defer os.Unsetenv("PEBBLE_ADMINS")
+
+	os.Setenv("PEBBLE_ADMINS", "")
+	uids, err := cli.GetAdditionalAdminUIDs()
+	c.Assert(err, IsNil)
+	c.Assert(uids, HasLen, 0)
+
+	os.Setenv("PEBBLE_ADMINS", "123")
+	uids, err = cli.GetAdditionalAdminUIDs()
+	c.Assert(err, IsNil)
+	c.Assert(uids, DeepEquals, []sys.UserID{123})
+
+	os.Setenv("PEBBLE_ADMINS", "123,345")
+	uids, err = cli.GetAdditionalAdminUIDs()
+	c.Assert(err, IsNil)
+	c.Assert(uids, DeepEquals, []sys.UserID{123, 345})
+
+	os.Setenv("PEBBLE_ADMINS", "123, 345")
+	_, err = cli.GetAdditionalAdminUIDs()
+	c.Assert(err, ErrorMatches, `cannot parse value " 345" in \$PEBBLE_ADMINS: strconv.ParseUint: parsing " 345": invalid syntax`)
```

Comment:
I think we should only do a partial match of the error string, not test the full thing. Perhaps:

```go
    c.Assert(err, ErrorMatches, `cannot parse value " 345".*`)
```